cmd/expiration-mailer/main_test.go

```diff
@@ -409,11 +409,10 @@ func TestFindCertsAtCapacity(t *testing.T) {
 	// The "48h0m0s" nag group should have its prometheus stat incremented once.
 	// Note: this is not the 24h0m0s nag as you would expect sending time.Hour
 	// * 24 to setup() for the nag duration. This is because all of the nags are
-	// offset by defaultNagCheckInterval, which is 24hrs.
+	// offset by Frequency, which is 24hrs.
```

Comment:
nit because I was confused here for a sec:
```suggestion
	// offset by Frequency, which is 24hrs in these tests.
```